File: src/main/java/me/miki/shindo/management/event/impl/EventRenderDamageTint.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;

public class EventRenderDamageTint extends Event {

    private final float partialTicks;

    public EventRenderDamageTint(float partialTicks) {
        this.partialTicks = partialTicks;
    }

    public float getPartialTicks() {
        return partialTicks;
    }
}

File: src/main/kotlin/com/shindoclient/shindo/management/event/impl/EventRenderDamageTint.kt
package com.shindoclient.shindo.management.event.impl

import com.shindoclient.shindo.management.event.Event

class EventRenderDamageTint(
    private val partialTicks: Float,
) : Event() {
    fun getPartialTicks(): Float = partialTicks
}
